### PR TITLE
Update ebpf comment format

### PR DESCRIPTION
**Pull request by sumit7754**
I have made the following changes in ebpf(backend):

Used triple slashes (///) for commenting functions and classes.
Used double slashes (//) for comments within the code.

### CODE REVIEW

**Comment by fruffy** [comment]
Thanks for doing this! It makes the entire compiler more consistent in style. 
Your PR is out-of-sync. You may have to rebase or merge.

**Comment by sumit7754** [comment]
You're welcome @fruffy  I'm glad to hear that it's making the compiler more consistent in style. If you have any more feedback then please let me know

File: backends/ebpf/target.h [hunk 1]
```diff
@@ -237,7 +237,7 @@ class P4TCTarget : public KernelSamplesTarget {
     }
 };
 
-// Target XDP
+/// Target XDP
```
Comment (fruffy) [inline backends/ebpf/target.h:240]:
```suggestion
/// Target XDP.
```

**Review by fruffy — CHANGES_REQUESTED** [review]

**Review by fruffy — APPROVED** [review]
There are still a bunch of lowercase instances and dots missing but I do not want to be too pedantic.

**Pull request merged**